src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
-export interface Figure {
+type Colors = 'red' | 'green' | 'blue';
+type Shapes = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  color: Colors;
+  shape: Shapes;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shapes;
 
-}
+  getArea(): number {
+    const semiPerimeter = (this.sideA + this.sideB + this.sideC) / 2;
+
+    return +Math.sqrt(Math.abs(semiPerimeter
+      * (semiPerimeter - this.sideA)
+      * (semiPerimeter - this.sideB)
+      * (semiPerimeter - this.sideC))).toFixed(2);
+  }
+
+  isInvalidShape(): boolean {
```

Comment:
Wow, cool method, I like it! Let's make it private, cause you are using it only internally

Reply:
'+'